proposals/0007-access-policy.md

```diff
@@ -0,0 +1,126 @@
+---
+simd: '0007'
+title: SIMD Access Policy
+authors:
+- Jacob Creech (Solana Foundation)
+category: Meta
+type: Meta
+status: Draft
+created: 2022-12-10
+---
+
+## Summary
+
+3 levels of access, in the order of increasing access:
+
+1. Triage
+
+    Requirement: One voucher from anyone with level 2 or above access.
+
+2. Core
+
+    Requirement: One voucher from anyone with level 3 access or two from
```

Comment:
I'd suggest that for Core access, you also first need Triage access. So Level 2 is like a promotion from level 1, you can't just start at Level 2

Reply:
Added promotion requirement.